src/main/java/io/supertokens/webserver/api/multitenancy/CreateOrUpdateConnectionUriDomainAPI.java

```diff
@@ -54,6 +56,32 @@ protected void doPut(HttpServletRequest req, HttpServletResponse resp) throws IO
         Boolean passwordlessEnabled = InputParser.parseBooleanOrThrowError(input, "passwordlessEnabled", true);
         JsonObject coreConfig = InputParser.parseJsonObjectOrThrowError(input, "coreConfig", true);
 
+        Boolean totpEnabled = null;
+        String[] firstFactors = null;
+        boolean hasFirstFactors = false;
+        String[] defaultRequiredFactorIds = null;
+        boolean hasDefaultRequiredFactorIds = false;
+
+        if (getVersionFromRequest(req).greaterThanOrEqualTo(SemVer.v4_1)) {
+            totpEnabled = InputParser.parseBooleanOrThrowError(input, "totpEnabled", true);
+            hasFirstFactors = input.has("firstFactors");
```

Comment:
need to throw a 400 in case of duplicate entries in this

Reply:
Done